acl/pom.xml

```diff
@@ -43,6 +39,14 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>rocketmq-srvutil</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.github.aliyun-mq</groupId>
+            <artifactId>rocketmq-slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.github.aliyun-mq</groupId>
+            <artifactId>rocketmq-logback-classic</artifactId>
+        </dependency>
```

Comment:
@RongtongJin Actually a more neutral repo(e.g. some other repos which belong to apache rocketmq) is considered, but we want to avoid the redundant polling procedure at present, because we'd better to keep up to date with the original version, current polling procedure is uncesscary.

On the other hand, I think current group id is ok because the License is more important, or we could refer to the dependency of fastjson.